src/native/corehost/fxr/fx_muxer.h

```diff
@@ -35,6 +35,9 @@ class fx_muxer_t
         host_context_t *context,
         coreclr_delegate_type delegate_type,
         void** delegate);
+    static int get_runtime_delegate_active(
```

Comment:
nit: I would prefer `get_runtime_delegate_for_active_context`

Reply:
I've recombined the variants back into `get_runtime_delegate`, so this one is gone.